app/src/main/java/at/aau/serg/adapters/JoinLobbyLobbiesAdapter.kt

```diff
@@ -0,0 +1,105 @@
+package at.aau.serg.adapters
+
+
+import android.content.ContextWrapper
+import android.content.Intent
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import android.widget.Toast
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.recyclerview.widget.RecyclerView
+import at.aau.serg.R
+import at.aau.serg.activities.LobbyActivity
+import at.aau.serg.logic.StoreToken
+import at.aau.serg.models.JoinLobbyLobby
+import at.aau.serg.models.LobbyJoin
+import at.aau.serg.network.CallbackCreator
+import at.aau.serg.network.HttpClient
+import com.google.gson.Gson
+import okhttp3.Response
+
+
+class JoinLobbyLobbiesAdapter(context: ContextWrapper, listdata: MutableList<JoinLobbyLobby>) :
```

Comment:
Can we also change the name to something like `LobbyAdapter`

Reply:
The name is debateble this version is more despriptive yours is shorter both would work for me